io/src/main/scala/laika/helium/generate/MergedCSSGenerator.scala

```diff
@@ -16,13 +16,19 @@
 
 package laika.helium.generate
 
-import cats.effect.Async
+import cats.effect.{Async, Sync}
 import cats.implicits._
 import laika.ast.Path.Root
-import laika.io.model.InputTree
+import laika.io.model.{BinaryInput, InputTree}
 
 private[helium] object MergedCSSGenerator {
 
+  private def merge[F[_]: Sync](inputs: Seq[BinaryInput[F]]): F[String] =
+    inputs.toList
+      .map(_.input.through(fs2.text.utf8.decode).compile.string)
+      .sequence
```

Comment:
```suggestion
      .traverse(_.input.through(fs2.text.utf8.decode).compile.string)
```

Also note the sync compiler used here.